### PR TITLE
Fixing doc errors and warnings pointed in #103.

### PR DESCRIPTION
**Pull request by tuopuu**
Fixed errors in #103. Documentation has to be recompiled after merging this.

### CODE REVIEW

**Comment by tuopuu** [comment]
@itziakos, could you please take a quick look before videocon? Is this a correct way of updating documentation, or should we recompile the whole thing when adding/modifying something?

File: doc/source/api/cuds.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Abstract CUDS interfaces
 .. autosummary::
 
    ~abstractmesh.ABCMesh
-   ~abstractparticles.ABCParticleContainer
+   ~abstractparticles.ABCParticle
```
Comment (nathanfranklin) [inline doc/source/api/cuds.rst]: ABCParticles

**Comment by tuopuu** [comment]
Thanks. Typo fixed now.

**Comment by nathanfranklin** [comment]
>  Is this a correct way of updating documentation, or should we recompile the whole thing when adding/modifying something?

yeah, looks good.    

one can just run the following command and see the results in the build directory to see if everything looks okay. 
``python setup.py build_sphinx``

**Comment by tuopuu** [comment]
Merged #133

**Pull request merged**